ZuluContent/Json/Converters/CreaturePropConverter.cs

```diff
@@ -0,0 +1,78 @@
+using Server.Mobiles;
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Server.Json
+{
+    public class CreaturePropConverterFactory : JsonConverterFactory
+    {
+        public override bool CanConvert(Type typeToConvert) => typeToConvert == typeof(CreatureProp);
+
+        public override JsonConverter CreateConverter(Type typeToConvert, JsonSerializerOptions options) =>
+            new CreaturePropConverter();
+    }
+
+    public class CreaturePropConverter : JsonConverter<CreatureProp>
+    {
+        public override CreatureProp Read(ref Utf8JsonReader reader, Type typeToConvert,
+            JsonSerializerOptions options)
+        {
+            double? min = null;
+            double? max = null;
+
+            switch (reader.TokenType)
+            {
+                case JsonTokenType.StartObject:
+                {
+                    reader.Read();
+                    while (reader.TokenType != JsonTokenType.EndObject)
+                    {
+                        if (reader.TokenType != JsonTokenType.PropertyName)
+                            throw new JsonException();
+
+                        var propertyName = reader.GetString()?.ToLowerInvariant();
+                        reader.Read();
+                        if (reader.TokenType != JsonTokenType.Number)
+                            throw new JsonException();
+
+                        if (propertyName == nameof(CreatureProp.Min).ToLowerInvariant())
+                            min = reader.GetDouble();
+                        else if (propertyName == nameof(CreatureProp.Max).ToLowerInvariant()) 
+                            max = reader.GetDouble();
+
+                        reader.Read();
+                    }
+
+                    if (!min.HasValue)
```

Comment:
Curious, should we check if max has a value as well?

Reply:
In this case it's optional to specify the max, it would make these two equivalent:

```json
{
    "damage": 12
}
// ...
{
    "damage": { "min": 12 }
}